src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -185,6 +185,18 @@ type DestinationDisplay {
   via: [String]
 }
 
+"Individual step of an elevation profile."
+type ElevationProfileStep {
+  "The horizontal distance from the start of the step, in meters."
+  distance: Float
+  """
+  The elevation at this distance, in meters.
```

Comment:
```suggestion
  The elevation at this distance, in meters above sea level.
```
I thought this was relative to the start of the profile, but if it it is in meters above sea level that is great!